src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -106,6 +108,16 @@ export default class UniversalDiscoveryModule extends Component {
         if (!!contentMeta && !isPreviewMetaReady) {
             this.props.loadContentInfo(this.props.restInfo, contentMeta.ContentInfo.Content._id, this.updateContentMetaWithCurrentVersion);
         }
+
+        if (!this.props.multiple) {
+            this.canSelectContent(contentMeta, (canSelectContent) => {
```

Comment:
Do you think it would be a good idea to extract the callback into a method like `setCanSelectContentState`?